.github/ISSUE_TEMPLATE/problem-description.md

```diff
@@ -8,37 +8,39 @@ about: Capture a concise, evidence-based problem
 2. Read the contributing guidelines.
 -->
 
-**Problem statement**
+## Problem statement
 
 <!-- Provide a concise, outcome-oriented summary that clearly states the core problem. 
 This section is about truly understanding the problem, not jumping to solutions. Use the "Who, What, When, Where, Why" framework to dig into the root cause. -->
 
-**Who is affected?**
+### What is the current state, unmet need, or symptom?
+<!-- Describe the situation users are facing today.-->
 
+### Who is affected?
 <!-- List specific personas, roles, or customer segments impacted. -->
 
-**When and where does this problem occur?**
+### When and where does this problem occur?
 
 <!-- Describe the timing or frequency of the problem, and what triggers it. Identify the context, processes, or conditions under which the problem arises. -->
 
-**Why does it matter?**
+### Why does it matter?
 
 <!-- Explain the underlying cause, not just the symptom. What's the business opportunity if we solve this? -->
 
-**Source of problem**
+## Trigger/Original Request
```

Comment:
I like title case for headlines. More importantly, our [style guide likes it too](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#title-case). 
But then why only use it for this headline while all the others are in sentence case? :D